zaza/openstack/charm_tests/cinder_lvm/tests.py

```diff
@@ -0,0 +1,136 @@
+#!/usr/bin/env python3
+
+# Copyright 2019 Canonical Ltd.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Encapsulate cinder-lvm testing."""
+
+import logging
+import uuid
+
+import zaza.model
+import zaza.openstack.charm_tests.test_utils as test_utils
+import zaza.openstack.utilities.openstack as openstack_utils
+
+
+def with_conf(application, config, model_name=None):
```

Comment:
you should use `BaseCharmTest.config_changed()` - https://github.com/freyes/zaza-openstack-tests/blob/master/zaza/openstack/charm_tests/test_utils.py#L264